assets/cohere.yaml

```diff
@@ -275,3 +305,96 @@
   monthly_active_users: unknown
   user_distribution: unknown
   failures: unknown
+  
+- type: application
+  name: Generate Endpoint
+  # General
+  organization: Cohere
+  description: >
+    This endpoint generates realistic text conditioned on a given input. 
+  url: https://docs.cohere.ai/reference/generate
+  # Construction
+  dependencies: [Cohere Base Model, Cohere Command Model]
+  adaptation: unknown
+  output_space: generation
+  # Downstream
+  access:
+    value: limited
+    
+- type: application
+  name: Embed Endpoint
+  # General
+  organization: Cohere
+  description: >
+    This endpoint returns text embeddings. An 
+    embedding is a list of floating point numbers that 
+    captures semantic information about the text that it represents. 
+  url: https://docs.cohere.ai/reference/embed
+  # Construction
+  dependencies: [Embed-English, Embed-Multilingual]
+  adaptation: unknown
+  output_space: embedding
+  # Downstream
+  access:
+    value: limited
+
+- type: application
+  name: Classify Endpoint
+  # General
+  organization: Cohere
+  description: >
+    This endpoint makes a prediction about which label best 
+    fits a specified text input. To make a prediction, 
+    Classify uses the provided examples of text + label 
+    pairs as a reference.
+  url: https://docs.cohere.ai/reference/classify
+  # Construction
+  adaptation: unknown
+  output_space: embedding
+  # Downstream
+  access:
+    value: limited
+
+- type: application
+  name: Summarize Endpoint
+  # General
+  organization: Cohere
+  description: >
+    This endpoint generates a succinct 
+    version of the original text that relays 
+    the most important information.
+  url: https://docs.cohere.ai/reference/summarize
+  # Construction
+  adaptation: unknown
+  output_space: generation
+  # Downstream
+  access:
+    value: limited
+  
+- type: application
```

Comment:
remove tokenize + detokenize - they are obviously relevant, but too low-level for the broader user base, so we are skipping for all models